homeassistant/components/kat_bulgaria/config_flow.py

```diff
@@ -0,0 +1,71 @@
+"""Config flow for KAT Bulgaria integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from kat_bulgaria.obligations import KatApi, KatErrorType
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+
+from .common import generate_entity_name
+from .const import CONF_DRIVING_LICENSE, CONF_PERSON_EGN, CONF_PERSON_NAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_FLOW_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_PERSON_NAME): str,
+        vol.Required(CONF_PERSON_EGN): str,
+        vol.Required(CONF_DRIVING_LICENSE): str,
+    }
+)
+
+
+class ConfigFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for kat_bulgaria."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+
+        # If no Input
+        if user_input is None:
+            return self.async_show_form(
+                step_id="user", data_schema=CONFIG_FLOW_DATA_SCHEMA
+            )
+
+        # Init user input values
+        user_egn = user_input[CONF_PERSON_EGN]
+        user_name = user_input[CONF_PERSON_NAME]
+        user_license_number = user_input[CONF_DRIVING_LICENSE]
+
+        # If this person (EGN) is already configured, abort
+        for entry in self._async_current_entries():
```

Comment:
Please either set a unique id for the config entry or use our helper to match for existing config entry data.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-ids

https://github.com/home-assistant/core/blob/95109072b523d47525421ee5f136a5ff23dd7e99/homeassistant/config_entries.py#L1520-L1527